samples/examples/income_classifier/infer-k8s.md

```diff
@@ -0,0 +1,298 @@
+## Tabular Income Classifier Kubernetes Test
+
+To run this notebook you need the inference data. This can be acquired in two ways:
+
+  * Run train.ipynb
+  * `gsutil cp -R gs://seldon-models/scv2/examples/income/infer-data .`
+
+```python
+import os
+os.environ["NAMESPACE"] = "seldon-mesh"
+```
+
+```python
+MESH_IP=!kubectl get svc seldon-mesh -n ${NAMESPACE} -o jsonpath='{.status.loadBalancer.ingress[0].ip}'
```

Comment:
:information_source: This is another case of not being able to catch everything (at least not reliably and robustly).  As before, I'd argue this is acceptable as it's a reasonably uncommon case and fairly obvious as to what's going on here and, thus, how to handle it.